### ISSUE TITLE
Add CC (Computercraft) Support

### ISSUE THREAD

**Issue by timuzkas** [thexaero/open-parties-and-claims#360]
We are trying to make ATM with CC, and for that we need to claim chunk for security reasons. Note, that this is server claim. but we found out, that turtle can be controlled with left-hannded pocket computer from CC, and take items from barrels, and chests.

Also, we cant click on monitors and use phones, but i can't figure out a flag for them

Thx!

**Comment by thexaero** [comment]
Is it specifically taking items that isn't protected?

You can create custom item groups with itemUseProtectionOptionalExceptionGroups that can be made toggleable in the player config. It's explained in the main config file. The same exists for blocks and entities.

**Comment by timuzkas** [comment]
It is inventories in chunks claimed by Open Parties and Claims.

**Comment by thexaero** [comment]
Yeah, sorry. I'm honestly confused by my own question. I think I had multiple issues opened and misunderstood what I was replying to. Surprised turtles aren't checking for block protection in this case because they supposedly have their owners' UUIDs so that they work with claim protection.
That being said, I'm wondering if the turtle's owner (the one who placed it) had access to the server claim or just the barrels/chests in the server claim. A turtle you place gets all your permissions.
